dev_guide/quota.adoc

```diff
@@ -87,116 +126,74 @@ in the system.
 
 === Compute Resources: Requests vs Limits
 
-When allocating compute resources, each container may specify a request and a limit value for either CPU or memory.  The quota tracking system will only cost the request value against the quota usage.  If a resource is tracked by quota, and no request value is provided, the associated entity is rejected as part of admission into the cluster.
-
-For an example, consider the following scenarios relative to tracking quota on CPU:
-
-.Quota tracking based on container requests
-[cols="3a,3a,3a,3a,8a",options="header"]
-|===
-
-|Pod |Container |Requests[CPU] |Limits[CPU] |Result
-
-|pod-x
-|a
-|100m
-|500m
-|The quota usage is incremented 100m
+When allocating compute resources, each container may specify a request and a limit value
+for either CPU or memory.  The quota can be configured to quota either value.  If the quota
+has a usage limit specified for requests.cpu or requests.memory, then it will require that 
```

Comment:
"usage limit" here makes me think of an actual limit, but this is about requests. It might be less confusing to say "If the quota has a value specified for requests.cpu or requests.memory..."